File: ssn/backend-spring5/src/main/java/com/giapyland/ssn/business/model/CommentDto.java
package com.giapyland.ssn.business.model;

import com.giapyland.ssn.data_access.Comment;

public class CommentDto {
	private String user;
	private long id;
	private long postId;
	private String content;
	private long timestamp;

	public String getUser() {
		return user;
	}

	public void setUser(String user) {
		this.user = user;
	}

	public long getId() {
		return id;
	}

	public void setId(long id) {
		this.id = id;
	}

	public long getPostId() {
		return postId;
	}

	public void setPostId(long postId) {
		this.postId = postId;
	}

	public String getContent() {
		return content;
	}

	public void setContent(String content) {
		this.content = content;
	}

	public long getTimestamp() {
		return timestamp;
	}

	public void setTimestamp(long timestamp) {
		this.timestamp = timestamp;
	}

	public CommentDto(Comment comment) {
		this.id = comment.getId();
		this.postId = comment.getPost().getId();
		this.user = comment.getUser();
		this.content = comment.getContent();
		this.timestamp = comment.getTimestamp();
	}
}

File: ssn/backend-dotnet/Ssn/Models/CommentDto.cs
using Ssn.DAL;

namespace Ssn.Models
{
    public class CommentDto
    {
        public int Id { get; set; }

        public string User { get; set; }

        public int PostId { get; set; }

        public string Content { get; set; }

        public long Timestamp { get; set; }

        public CommentDto(int postId, Comment comment)
        {
            Id = comment.Id;
            User = comment.User;
            PostId = postId;
            Content = comment.Content;
            Timestamp = comment.Timestamp;
        }
    }
}